Tasks/SonarQubePostTest/PRCA/PostComments-Module.psm1

```diff
@@ -37,11 +101,10 @@ function LoadTfsClientAssemblies
         "Microsoft.VisualStudio.Services.CodeReview.Discussion.WebApi.dll",    
         "Microsoft.VisualStudio.Services.CodeReview.WebApi.dll",        
         "Microsoft.VisualStudio.Services.Common.dll",
-        "Microsoft.VisualStudio.Services.WebApi.dll",
-        "Newtonsoft.Json.dll")
```

Comment:
I'm not actually deserializing from a json file since the ReportProcessor module will return the data directly.